qcodes/dataset/data_set.py

```diff
@@ -849,47 +849,6 @@ def _validate_parameters(*params: Union[str, ParamSpec, _BaseParameter]
                 valid_param_names.append(maybeParam)
         return valid_param_names
 
-    @deprecate('This method does not accurately represent the dataset.',
-               'Use `get_parameter_data` instead.')
-    def get_data(self,
```

Comment:
we also need to remove `get_values` and `get_setpoints`

Reply:
+1 for get_values but get_setpoints is not deprecated :( It probably shuld have been

Reply:
> It probably shuld have been

true. but it's ok, i guess.